index.html

```diff
@@ -289,7 +289,114 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+       <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form  id ="form" action="https://formspree.io/f/mwkakrqj" method="GET">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            required
+          />
+          <div class="name-form">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              placeholder="First name"
+              required
+            />
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              placeholder="Last name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <input
+            type="email"
+            id="email"
+            name="_replyto"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+            required
+          ></textarea>
```

Comment:
Good job for adding the `required` attribute for validation, so make sure to validate that it's not over `500` characters long.